### DISCUSSION TITLE
Are there special characters in any of the quotes?

### DISCUSSION THREAD

**Discussion by someunknowndude** [lukePeavey/quotable#116] [Questions]
So far I haven't seen any quotes pop up that e.g. have letters with accents (è) or umlaute (ö,ü,ä)

If these do exist in some quotes however, is there a way to filter them out directly instead of having to match each quote with all the special characters commonly used in other languages and requesting a new quote if one was found?

**Comment by lukePeavey** [comment]
**quotes**
The quotes themselves do not contain any non-latin characters.  

**Author names**
We use the exact name and spelling on the author's wikipedia page, so many of the names **do** contain non-latin characters like the ones mentioned in your question.  It's also possible that an author's `bio` will contain non-latin characters (for example, if the author's name is mentioned in the bio). 

Regarding the second part of your question, the API does not currently provide any way to filter out non-latin characters.
